ci-operator/step-registry/openshift/agentic/trt/eval/judge/openshift-agentic-trt-eval-judge-commands.sh

```
@@ -0,0 +1,344 @@
+
+# =============================================
+# SUMMARY
+# =============================================
+echo ""
+echo "--- Summary ---"
+echo "Checks: ${CHECKS_PASS}/${CHECKS_TOTAL} passed"
+
+# Print file comparison
+echo ""
+echo "Files changed by Claude:"
+echo "${CLAUDE_FILES}" | sed 's/^/  /' || echo "  (none)"
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)